README.md

```diff
@@ -319,6 +319,41 @@ td.fetchUserSegments(token, successCallback, errorCallback)
 ```
 *N.B.* This feature is not enabled on accounts by defaut, please contact support for more information.
 
+### Treasure#fetchUserSegments(options, success, failure)
+
+**Parameters:**
+
+* **options** : Object (required) - User Segment object 
```

Comment:
Is the string token deprecated or should we document it?

Reply:
string token is stills supported (and documented above) this is just the variation where the first parameter is an object instead

Reply:
Ah, sorry! I missed that